src/themes/money/theme.json

```diff
@@ -138,102 +146,80 @@
   },
 
   "----------------------------- Variants": "-----------------------------",
-  "modules": {
-
-    "content-hero-module": {
-      "h1": {
-        "mb": "sm",
-        "mt": 0,
-        "pt": ["sm", "sm", "lg"]
-      },
-
-      "h4": { "fontSize": "md", "fontWeight": "normal" },
-
-      "p": {
-        "width": ["auto", "grid.md.5", "grid.lg.5"],
-        "textAlign": ["left", "left", "center"],
-        "my": [0, 0, "xxs"],
-        "px": [0, 0, "md"],
-        "display": ["inline", "inline", "block"]
-      },
-
-      "a": {
-        "fontSize": ["base", "base", "lg"],
-        "textDecoration": "underline"
-      },
-
-
-      "content": {
-        "display": "flex",
-        "alignItems": "flex-end",
 
-        "modules": {
-          "width": ["100%", "50%", "50%"],
-          "mb": "lg"
-        },
+  
+  "accordion": {
+    "button": {
+      "padding": "sm",
+      "bg": ["grey-05", "transparent"],
+      "fontFamily": "base",
+      "fontSize": "base",
+      "fontWeight": "bold",
+      "color": "textColor",
+      "alignItems": "center",
+      "borderWidth": "0 0 1px 0",
+      "borderStyle": "solid",
+      "borderColor": "grey-20"
+    },
+    "caret": {
+      "color": "cannon-pink"
+    },
 
-        "image": {
-          "height": [0, 0, 320],
-          "width": [0, 0, "50%"],
-          "mx": "auto",
-          "backgroundSize": 500,
-          "backgroundPosition": "bottom"
-        }
+    "isActive": {
+      "caret": {
+        "color": "cranberry"
       }
     }
-
   },
 
-  "compounds": {
-
-    "card": {
-      "display": "flex",
-      "width": ["100%", "49%", "grid.lg.3"],
-      "flexDirection": "column",
-      "borderRadius": "sm",
-      "borderStyle": "solid",
-      "borderColor": "dark-2",
-      "borderWidth": "1px",
-      "mb": "sm",
-      "img": {
-        "sm": { "width": "100%", "height": 230 },
-        "md": { "width": "100%", "height": 230 },
-        "lg": { "maxWidth": "grid.lg.3", "height": 230 }
+  "sideNav": {
+    "internalLink": {
+      "color": "grey-60",
+      "padding": "5px 0 5px 35px",
+      "after": {
+        "content":"\"\"",
+        "position":"absolute",
```

Comment:
layout stuff is best in the component imo, otherwise it's just duplicated across themes

Reply:
talked irl, this is fine 👍